packages/design-system/adr/process-ds-transformation.md

```diff
@@ -0,0 +1,87 @@
+# Transform Design system process
+
+## Current status of DS?
+
+- A dedicated team → 2 DS dev + 1 DS designer
+- In _collaboration_ with products designers/content designers
+- _Lack of collaboration_ from product developers
+- _Low usage_ in products
+- How can we move from the DS team to something?
+
+## Org proposals
+
+- Have a composite team from the different domains (1 or 2 devs per domain) and Designers => `Design System Champions 🎖`
+
+  - Goal : Bring the product teams inside the process of build the DS
+
+  - Existing teams (nb of frontend dev):
+
+    - API : Tester / Designer / Creator (1,5?)
+    - DGD : Data Preparation / Data Stewardship / Data Inventory (4,5)
+    - DPD : Pipeline Designer (3)
+    - TDOPS : TMC / Portal / IDAAS (7)
+    - STITCH ?
```

Comment:
If they do not plan to consume the DS, I would not bother. Dustin has been helping sometimes on the design side, but Stitch has its own component librairies and remain ever so wary of merging things.